main.go

```diff
@@ -285,3 +300,7 @@ func isMultipleModes(bools ...bool) bool {
 
 	return false
 }
+
+func prefixRoot(root, path string) string {
+	return pathutil.Clean(fmt.Sprintf("/%s/%s", root, path))
```

Comment:
This should be path. I think that's why its not building.

Reply:
Whoops, I forgot to add the import alias.

Can't use path, it's already used.

Reply:
Tried a couple commits to get it going. Feel free to squash or whatever @tonglil

Reply:
Actually the build is working. Looks like the merge commit might have been stale.